applications/RANSApplication/python_scripts/formulations/turbulence_models/k_omega_rans_formulation.py

```diff
@@ -0,0 +1,142 @@
+# import kratos
+import KratosMultiphysics as Kratos
+
+# import RANS
+import KratosMultiphysics.RANSApplication as KratosRANS
+
+# import formulation interfaces
+from KratosMultiphysics.RANSApplication.formulations.turbulence_models.scalar_turbulence_model_rans_formulation import ScalarTurbulenceModelRansFormulation
+from KratosMultiphysics.RANSApplication.formulations.turbulence_models.two_equation_turbulence_model_rans_formulation import TwoEquationTurbulenceModelRansFormulation
+
+# import utilities
+from KratosMultiphysics.RANSApplication import RansCalculationUtilities
+
+class KOmegaKRansFormulation(ScalarTurbulenceModelRansFormulation):
+    def GetSolvingVariable(self):
+        return KratosRANS.TURBULENT_KINETIC_ENERGY
+
+    def GetElementNamePrefix(self):
+        return "RansKOmegaK"
+
+    def GetConditionNamePrefix(self):
+        return ""
+
+
+class KOmegaOmegaRansFormulation(ScalarTurbulenceModelRansFormulation):
+    def GetSolvingVariable(self):
+        return KratosRANS.TURBULENT_SPECIFIC_ENERGY_DISSIPATION_RATE
+
+    def GetElementNamePrefix(self):
+        return "RansKOmegaOmega"
+
+    def GetConditionNamePrefix(self):
+        return "RansKOmegaOmega"
+
+
+class KOmegaRansFormulation(TwoEquationTurbulenceModelRansFormulation):
+    def __init__(self, model_part, settings):
+        default_settings = Kratos.Parameters(r'''
+        {
+            "formulation_name": "k_Omega",
```

Comment:
```suggestion
            "formulation_name": "k_omega",
```
Snake case convention (up to you)

Reply:
it was a mistake.... now changed :)